packages/core/src/wallets/keplr-walletconnect.ts

```diff
@@ -1,10 +1,11 @@
 import { IKeplrWalletConnectV1, Wallet } from '../types'
+import imageUrl from './images/keplr-walletconnect.png'
```

Comment:
@ebaker @NoahSaso only issue is here, we need to figure out how to type this in typescript... I realized we cannot use the ts-ignore here 😅